great_expectations/rule_based_profiler/types/data_assistant_result/plot_components.py

```diff
@@ -102,18 +145,22 @@ def determine_plot_title(
 
     Args:
         metric_plot_component: Plot utility corresponding to a given metric.
+        batch_plot_component: Plot utility corresponding to a given batch.
         domain_plot_component: Plot utility corresponding to a given domain.
 
     Returns:
         An Altair TitleParam object
 
     """
-    contents: str = f"{metric_plot_component.title} per {domain_plot_component.title}"
+    contents: str = f"{metric_plot_component.title} per {batch_plot_component.title}"
     subtitle: Optional[str] = domain_plot_component.subtitle
+    domain_selector: Optional[bool] = bool(domain_plot_component.name)
```

Comment:
If we're casting with `bool`, is this just `bool` instead of `Optional[bool]`?

Reply:
🤦🏻

Reply:
I just changed it to this: `domain_selector: Optional[str] = domain_plot_component.name`